sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs

```diff
@@ -38,6 +42,8 @@ public DataFeed(string dataFeedName, DataFeedSource dataSource, DataFeedGranular
             Granularity = dataFeedGranularity;
             Schema = dataFeedSchema;
             IngestionSettings = dataFeedIngestionSettings;
+            Administrators = new ChangeTrackingList<string>();
```

Comment:
Could you remind me what `ChangeTrackingList` is?

Reply:
It's a helper class defined in Core and used by CodeGen when dealing with collections (they have a `ChangeTrackingDictionary` as well). It's basically a `List`, but it behaves a bit differently during serialization.

Initializing Administrators with an empty `ChangeTrackingList<string>`:
```cs
var dataFeed = new DataFeed(...);
await client.CreateDataFeed(dataFeed); // user didn't change 'Administrators', so we don't send it to the service.
```

Initializing Administrators with an empty `List<string>`:
```cs
var dataFeed = new DataFeed(...);
await client.CreateDataFeed(dataFeed); // user didn't change 'Administrators', but we send an empty list anyway.
```

CodeGen uses a `ChangeTrackingList` to identify when the user has intentionally updated the list, deciding whether or not it should be included in the HTTP request body. It's specially useful when you have nullable non-required properties, a scenario in which .NET cannot differentiate between `undefined` and `null`.